src/logging.c

```diff
@@ -414,6 +414,92 @@ static const char * copy_from_orig(
   return logging_output->buffer;
 }
 
+// copy buffers and decode the escape characters if they're exists
+static const char * decode_orig(
+  const logging_input_t * logging_input,
+  rcutils_char_array_t * logging_output,
+  size_t start_offset, size_t end_offset)
+{
+  size_t back_slash_index = rcutils_findn(
+    g_rcutils_logging_output_format_string + start_offset, '\\', end_offset - start_offset);
```

Comment:
probably we should declare `\\` const char variable or definition?

Reply:
Thanks. Updated in https://github.com/ros2/rcutils/pull/426/commits/097c219af9f5b382614aa270d316f278d5051a21